hibike/devices/YogiBear/LED.cpp

```diff
@@ -19,6 +19,9 @@ void ctrl_RED() {
 	} else {
 		digitalWrite(LED_RED, LOW);
 	}
+	if(!isMotorEnabled()) {
```

Comment:
Compacted all LED functionality into ctrl_LEDs(). Used booleans to control state and wrote once at the end.